draft-packages/collapsible/package.json

```diff
@@ -32,6 +32,7 @@
     "@kaizen/component-base": "^1.1.1",
     "@kaizen/component-library": "^16.1.1",
     "@kaizen/typography": "^2.3.2",
+    "@kaizen/button": "^1.3.3",
```

Comment:
You may want to move this up so it's in the correct alphabetical order, otherwise things will move around when anyone uses a command to add a package.